### DISCUSSION TITLE
Setting GitHub Pages commit deployments to a bot user

### DISCUSSION THREAD

**Discussion by sumanthratna** [upptime/upptime#212] [Q&A] [answered]
Right now, I have a personal access token set up for `GH_PAT`. It works great (https://github.com/HackTJ/status), except that commits to `gh-pages` are under my username, since I created the PAT. 

I would prefer a bot user made the commits instead, such as `github-actions[bot]`. I think this could make configurations easier, since Upptime could then use `$GITHUB_TOKEN` (which is available by default I think) instead of `$GH_PAT`. 

Is there an easy way to do this right now? If not, consider this a feature request

edit: linking https://github.com/upptime/upptime/issues/118, which may be related

**Comment by AnandChowdhary** [comment, marked as answer]
Closing as a duplicate of https://github.com/upptime/upptime/discussions/211.
